Makefile

```diff
@@ -190,6 +204,26 @@ nxMySQL:
 	mkdir -p ../../release; \
 	mv $@_$${VERSION}.zip ../../release/
 
+nxOMSAgent:
+	rm -rf output/staging; \
+	VERSION="1.0"; \
+	PROVIDERS="nxOMSAgent nxOMSCustomLog nxOMSPlugin nxOMSSyslog nxService nxPackage"; \
```

Comment:
We will move the nxOMSCustomLog and nxOMSPlugin out into separate modules, so that they can be upgraded separately.